README.md

```diff
@@ -48,106 +47,121 @@ To get a personal developer edition of RHEL for SAP solutions, please register a
  product but only a special bundling. The subscription grants you access to the additional
  packages through our content delivery network (CDN) after installation.
 
-For supported RHEL releases [click here](https://access.redhat.com/solutions/2479121)
+For supported RHEL releases [click here](https://access.redhat.com/solutions/2479121).
 
 It is also important that your disks are setup according to the [SAP storage requirements for SAP HANA](https://www.sap.com/documents/2015/03/74cdb554-5a7c-0010-8F2c7-eda71af511fa.html). This [BLOG](https://blogs.sap.com/2017/03/07/the-ultimate-guide-to-effective-sizing-of-sap-hana/) is also quite helpful when sizing HANA systems.
 You can use the [disk-init](https://galaxy.ansible.com/mk-ansible-roles/disk-init/) role to automate this process
 
-If you want to use this system in production make sure time service is configured correctly. You can use [rhel-system-roles](https://access.redhat.com/articles/3050101) to automate this
+If you want to use this system in production, make sure that the time service is configured correctly. You can use [rhel-system-roles](https://access.redhat.com/articles/3050101) to automate this.
+
+Note
+----
+For finding out which SAP notes will be used by this role, please check the contents of variable __sap_hana_preconfigure_sapnotes in files vars/*.yml (choose the file which matches your OS distribution and version)
+
+Please do not run this role against a productive SAP HANA system. The role will unconditionally make changes to the managed node, which might not be intended on productive SAP HANA systems.
```

Comment:
"production" not "productive"

Reply:
I rephrased the warning in https://github.com/berndfinger/sap-hana-preconfigure/commit/ce162cb8073e086ab19ffd37b3a808b070c7cfaf .